variables.tf

```
@@ -169,6 +169,12 @@ variable "policy_document" {
   type        = string
 }
 
+variable "requires_compatibilities" {
+  default     = ["EC2", "FARGATE"]
```

```suggestion
  default     = ["FARGATE"]
```
shall we use `FARGATE` as default here?